project4/project/ebs/static/ebs/css/manage_blog.css

```diff
@@ -1,3 +1,71 @@
+h1, h2, h3, h4, h5{
+	font-family: 'Roboto',sans-serif;
+}
 .top-margin{
 	margin-top:30px;
+}
+.search{
+	width: 300px;
+    margin-left:150px;
+}
+.image{
+	max-width: 180px;
+	max-height:150px;
+}
+#delete-button{
+	margin-left:30px;
+}
+.text{
+	font-size: 13px;
+    color: #777;
+    margin-top: 9px;
```

Comment:
add "white-space:normal;" to resolve table width issue on small screen resolution.